### DISCUSSION TITLE
Automatically selecting first result

### DISCUSSION THREAD

**Discussion by digitalade** [TarekRaafat/autoComplete.js#361] [Q&A]
What would be the recommended way of automatically highlighting result #1 in the drop down?

Therefore when enter key is pressed, the selection event will fire for that first result?

**Comment by Arnaud34000** [comment]
`events: {
    input: {
        keydown: (event) =>{
            if(event.key == 'Enter'){
                autoCompleteJS.select(0);
            }
        }
    }
}`

**Reply by andrewgtibbetts** [reply]
Heads up: this will override the built-in / default `keydown` listeners (`ArrowUp`, `ArrowDown`, `Return`, `Tab` and `Esc`). I switched to `keyup` which isn't used (input has listeners on `input`, `keydown` and `blur`, list listens on `mousedown`, and `click`) and it worked fine.
